src/components/itineraries/DraggableItineraryItem.tsx

```diff
@@ -62,6 +70,8 @@ const DraggableItineraryItem: FC<Props> = ({ itineraryItem, index }) => {
     }
   )
 
+  const onCheck= () => setChecked(id, !checked)
```

Comment:
space after `onCheck`. Let's add this to ESLint.

Reply:
Linter now catches these things